flask_resty/testing.py

```diff
@@ -97,16 +121,7 @@ def assert_response(response, expected_status_code, expected_data=UNDEFINED):
     else:
         response_data = get_errors(response)
 
-    assert_shape(response_data, expected_data)
-
+    if not isinstance(expected_data, Predicate):
```

Comment:
I am not sure I fully understand the purpose of these two lines. then you should just be able to do `assert response_data = expected_data` then, instead of `assert_shape(response_data, expected_data)`. no?